app/views/catalog/_show_collection.html.erb

```diff
@@ -15,17 +15,17 @@
           </li>
           <li class='nav-item flex-fill' role='tab'>
             <a class='nav-link ' data-toggle='tab' href='#description' role='tab'>
-              Collection Description
+              Collection Description & Creator Information
```

Comment:
Should this instead use `&amp;` or `&#x00026;`?

Reply:
I don't think you need to escape it just for HTML display do you?